apps/blockchain/lib/blockchain/application.ex

```diff
@@ -1,5 +1,5 @@
 defmodule Blockchain.Application do
-  # See http://elixir-lang.org/docs/stable/elixir/Application.html
+  # See https://elixir-lang.org/getting-started/mix-otp/supervisor-and-application.html
```

Comment:
I think we can remove the comments from this module. They are auto-generated when created the `mix` project.